crates/router/src/core/webhooks.rs

```diff
@@ -961,17 +961,25 @@ pub async fn webhooks_core<W: types::OutgoingWebhookType, Ctx: PaymentMethodRetr
             return Ok((response, WebhookResponseTracker::NoEffect));
         }
     };
+    logger::info!(event_type=?event_type);
 
-    let process_webhook_further = utils::lookup_webhook_event(
+    let is_webhook_event_not_event_not_supported = !matches!(
```

Comment:
Minor nitpick. Confused me for a moment.
```suggestion
    let is_webhook_event_supported = !matches!(
```